### PR TITLE
Users/tolee/new api version

### PR DESCRIPTION
**Pull request by tonykslee**
Reposting this pr: https://github.com/Azure/azure-rest-api-specs/pull/13524

Since the first commit of that PR did not contain the existing unmodified file versions

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [x] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by kairu-ms — CHANGES_REQUESTED** [review]
@tonykslee I'm trying to fix issues in previous workbook api version. https://github.com/Azure/azure-rest-api-specs/pull/13951#issuecomment-820248090. Please fix them in new api version first. Thanks.

**Review by kairu-ms — CHANGES_REQUESTED** [review]
Please rename definitions in workbooks_api as well.

File: specification/applicationinsights/resource-manager/readme.md [hunk 1]
```diff
@@ -216,6 +216,18 @@ These settings apply only when `--tag=package-preview-2021-03-only` is specified
...
+
+These settings apply only when `--tag=package-2021-03-08` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-03-08'
+input-file:
+  - Microsoft.Insights/stable/2021-03-08/myworkbooks_API.json
+  - Microsoft.Insights/stable/2021-03-08/workbooks_API.json
+  - Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json
+```
+
```
Comment (kairu-ms) [inline specification/applicationinsights/resource-manager/readme.md:230]:
```suggestion
### Tag: package-2021-03-08-only

These settings apply only when `--tag=package-2021-03-08-only` is specified on the command line.

``` yaml $(tag) == 'package-2021-03-08-only'
input-file:
  - Microsoft.Insights/stable/2021-03-08/myworkbooks_API.json
  - Microsoft.Insights/stable/2021-03-08/workbooks_API.json
  - Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json
```

```

Comment (tonykslee) [reply specification/applicationinsights/resource-manager/readme.md]: what does the -only suffix of the tag do? quick search in swagger wiki and github didn't show anything relevant.

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Insights/operations": {
```
Comment (kairu-ms) [inline specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json:37]: This URL is for operations of ai. It's not only for workbook Operations. The supports api-versions are  '2014-04-01,2014-06-01,2015-04-01'.
image

**Comment by kairu-ms** [comment]
Please link examples in API definition.

File: specification/applicationinsights/resource-manager/readme.md (hunk 1 above)
Comment (kairu-ms) [reply specification/applicationinsights/resource-manager/readme.md]: if a tag contains only part of API files, it should add `-only`.

**Comment by chiragg4u** [comment]
Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
You will need to close this PR and reopen new PR with above recommendation to make progress.

**Comment by tonykslee** [comment]
> Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
> You will need to close this PR and reopen new PR with above recommendation to make progress.

we made this mistake on the original PR (https://github.com/Azure/azure-rest-api-specs/pull/13524) , so I created this new PR where the first commit contains the original files (9e2b9e5d88fe9fcdb879aa23e745d40bbbd10883). Then I merged in the changes from the previous PR. So you should still be able to compare the current version to that "first commit" version.

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json (hunk 2 above)
Comment (tonykslee) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json:37]: the Microsoft.Insights RP is huge and is maintained by many teams. The operations api does not include the sub apis' api-version. So you can just use the 2015-04-01 api-version. 

The newly added endpoint "/Workbooks/Revisions/Read" was merged into the RP yesterday: https://msazure.visualstudio.com/One/_git/AzureUX-Insights-MonRP/pullrequest/4386680

**Comment by chiragg4u** [comment]
The Revisions API changes looks OK. Marking it ARMSignedoff. Please resolve warning and follow the process given @ https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review to complete and get the PR merged

**Comment by tonykslee** [comment]
> Please link examples in API definition.

@kairu-ms  Looks like all examples are referenced in the api-definition. Please review/merge when you get the chance

**Comment by raych1** [comment]
@jianyexi , do you have any idea of why `cross version breakingchange` take an old preview version as base for comparison?

**Comment by raych1** [comment]
> @jianyexi , do you have any idea of why `cross version breakingchange` take an old preview version as base for comparison?

Ignore this comparison error because the other breaking change check against previous stable version passed.

**Review by raych1 — APPROVED** [review]

**Comment by tonykslee** [comment]
Thank you @raych1
whenever ready, this PR can be merged as well.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**